.babelrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = api => ({
   presets: [
+    '@babel/preset-typescript',
```

Comment:
ooh so maybe that's why it was using import `syntax`! are yo fixing upstream?

Reply:
no :/ this didn't have an effect